packages/jsii-pacmak/test/expected.jsii-calc/kotlin/src/main/kotlin/software/amazon/jsii/tests/calculator/Add.kt

```diff
@@ -0,0 +1,40 @@
+package software.amazon.jsii.tests.calculator
+
+/**
+ * The "+" binary operation.
+ */
+@javax.annotation.Generated("jsii-pacmak")
+@software.amazon.jsii.Jsii(module = software.amazon.jsii.tests.calculator.`$Module`::class, fqn = "jsii-calc.Add")
+@software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+open class Add : software.amazon.jsii.tests.calculator.BinaryOperation {
```

Comment:
Does JSII have a concept of immutable types? e.g. objects that cannot be extended?

Reply:
TypeScript doesn't have such a thing, so no, we don't have that. We may come up with clever tricks to make ti possible in the future though (this is an idea that we have already touched on several times in the past - for there are really interesting aspects to it).